modules/auth/cms_authentication_email_service_parameters.e

```diff
@@ -101,10 +107,23 @@ feature	-- Access
 	contact_email: IMMUTABLE_STRING_8
 			-- Contact email.
 
+	site_name: IMMUTABLE_STRING_8
+			-- Site name.
```

Comment:
you could precise it is UTF-8 encoded
maybe in the comment or in the name

utf_8_site_name: ...